tools/scylla-sstable.cc

```diff
@@ -258,7 +258,15 @@ const std::vector<sstables::shared_sstable> load_sstables(schema_ptr schema, sst
         data_dictionary::storage_options local;
         auto sst = sst_man.make_sstable(schema, dir_path.c_str(), local, ed.generation, sstables::sstable_state::normal, ed.version, ed.format);
 
-        co_await sst->load(schema->get_sharder(), sstables::sstable_open_config{.load_first_and_last_position_metadata = false});
+        try {
+            co_await sst->load(schema->get_sharder(), sstables::sstable_open_config{.load_first_and_last_position_metadata = false});
+        } catch (...) {
+            // Print each individual error here since parallel_for_each
+            // will propagate only one of them up the stack.
+            auto msg = fmt::format("Could not load SSTable: {}", sst->get_filename());
+            fmt::print(std::cerr, "{}: {}", msg, std::current_exception());
```

Comment:
You need to add a new-line because `fmt::print()` doesn't add one.

Reply:
oh, right :-/